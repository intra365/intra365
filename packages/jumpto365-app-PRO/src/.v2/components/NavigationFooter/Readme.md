
# TODO: Make sample


```js
<NavigationFooter about="🍕" />
```