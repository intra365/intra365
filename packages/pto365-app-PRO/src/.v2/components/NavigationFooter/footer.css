.NavigationFooter {
    background-color: #2a7ab9bb;
    color: #dddddd;
    min-height: 200px;
    
}

.NavigationFooterContent{
 padding-top: 20px;
 padding-left: 20px;
 padding-right: 20px;

 padding-bottom: 20px;

}


.NavigationFooter a:visited,
.NavigationFooter a{
    
    color: #dddddd;
}

.NavigationFooter a:hover{
    
    color: #dddddd;
    font-weight: bolder
}

