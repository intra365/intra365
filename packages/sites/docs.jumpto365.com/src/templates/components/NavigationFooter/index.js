/**
 * Copyright (c) 2018-present, jumpto365, Inc.
 *
 * @emails jumpto365
 */

import NavigationFooter from './NavigationFooter';

export default NavigationFooter;
