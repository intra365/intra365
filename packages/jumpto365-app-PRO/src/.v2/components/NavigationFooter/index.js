import React, { Component } from 'react';
import PropTypes from 'prop-types'
import "./footer.css"
import VersionId from '../Component';


/**
 * Describe overall purpose of the component
 *
 * @export
 * @class NavigationFooter
 * @extends {Component}
 */
export default class NavigationFooter extends Component { 

    static propTypes  = {
        about : PropTypes.string   // sample - remove or rename
    }
    

    constructor(props) {
        super(props);
    }

    /**
     * Required method return the output of the component
     *
     * @returns
     * @memberof NavigationFooter
     */
    render() {
        return (

            <div className="NavigationFooter">
                
            <div className="NavigationFooterContent ms-font-m"  >
            <div style={{float:"right"}}>

                Version <VersionId />
        </div>
       
  
            <div >
                Orginal Design ©2017-18&nbsp;
                <a href="https://twitter.com/thatmattwade" rel="noopener noreferrer" target="_blank">Matt Wade</a>&nbsp;&amp;&nbsp; 
                <a href="http://icansharepoint.com" rel="noopener noreferrer" target="_blank">icansharepoint.com</a>&nbsp; Web Development ©2017-18 &nbsp;
                <a href="https://twitter.com/niegrejoh" rel="noopener noreferrer" target="_blank">Niels Gregers Johansen</a> &nbsp;
                <a href="https://www.hexatown.com" rel="noopener noreferrer" target="_blank">hexatown.com</a>  &nbsp;
                </div>
        


        </div>
        </div>

        )
    }
}

