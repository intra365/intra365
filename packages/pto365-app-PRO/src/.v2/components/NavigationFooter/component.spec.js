import React, { Component } from 'react';

import NavigationFooter from '.'

test('Check that it renders', () => {

    const wrapper = shallow(<NavigationFooter about="🍕"/>);

    expect(wrapper).toMatchSnapshot();
 
}

)