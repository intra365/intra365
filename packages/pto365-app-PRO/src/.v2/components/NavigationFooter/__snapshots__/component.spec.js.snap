// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check that it renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavigationFooter
    about="🍕"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="NavigationFooterContent ms-font-m"
>
        <div
                style={
                        Object {
                                "float": "right",
                              }
                }
        >
                Version 
                <VersionId />
        </div>
        <div>
                Orginal Design ©2017-18 
                <a
                        href="https://twitter.com/thatmattwade"
                        rel="noopener noreferrer"
                        target="_blank"
                >
                        Matt Wade
                </a>
                 & 
                <a
                        href="http://icansharepoint.com"
                        rel="noopener noreferrer"
                        target="_blank"
                >
                        icansharepoint.com
                </a>
                  Web Development ©2017-18  
                <a
                        href="https://twitter.com/niegrejoh"
                        rel="noopener noreferrer"
                        target="_blank"
                >
                        Niels Gregers Johansen
                </a>
                  
                <a
                        href="https://www.hexatown.com"
                        rel="noopener noreferrer"
                        target="_blank"
                >
                        hexatown.com
                </a>
                   
        </div>
</div>,
      "className": "NavigationFooter",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            style={
                        Object {
                                    "float": "right",
                                  }
            }
>
            Version 
            <VersionId />
</div>,
          <div>
            Orginal Design ©2017-18 
            <a
                        href="https://twitter.com/thatmattwade"
                        rel="noopener noreferrer"
                        target="_blank"
            >
                        Matt Wade
            </a>
             & 
            <a
                        href="http://icansharepoint.com"
                        rel="noopener noreferrer"
                        target="_blank"
            >
                        icansharepoint.com
            </a>
              Web Development ©2017-18  
            <a
                        href="https://twitter.com/niegrejoh"
                        rel="noopener noreferrer"
                        target="_blank"
            >
                        Niels Gregers Johansen
            </a>
              
            <a
                        href="https://www.hexatown.com"
                        rel="noopener noreferrer"
                        target="_blank"
            >
                        hexatown.com
            </a>
               
</div>,
        ],
        "className": "NavigationFooterContent ms-font-m",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Version ",
              <VersionId />,
            ],
            "style": Object {
              "float": "right",
            },
          },
          "ref": null,
          "rendered": Array [
            "Version ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Orginal Design ©2017-18 ",
              <a
                href="https://twitter.com/thatmattwade"
                rel="noopener noreferrer"
                target="_blank"
>
                Matt Wade
</a>,
              " & ",
              <a
                href="http://icansharepoint.com"
                rel="noopener noreferrer"
                target="_blank"
>
                icansharepoint.com
</a>,
              "  Web Development ©2017-18  ",
              <a
                href="https://twitter.com/niegrejoh"
                rel="noopener noreferrer"
                target="_blank"
>
                Niels Gregers Johansen
</a>,
              "  ",
              <a
                href="https://www.hexatown.com"
                rel="noopener noreferrer"
                target="_blank"
>
                hexatown.com
</a>,
              "   ",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Orginal Design ©2017-18 ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Matt Wade",
                "href": "https://twitter.com/thatmattwade",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": "Matt Wade",
              "type": "a",
            },
            " & ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "icansharepoint.com",
                "href": "http://icansharepoint.com",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": "icansharepoint.com",
              "type": "a",
            },
            "  Web Development ©2017-18  ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Niels Gregers Johansen",
                "href": "https://twitter.com/niegrejoh",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": "Niels Gregers Johansen",
              "type": "a",
            },
            "  ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "hexatown.com",
                "href": "https://www.hexatown.com",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": "hexatown.com",
              "type": "a",
            },
            "   ",
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="NavigationFooterContent ms-font-m"
>
          <div
                    style={
                              Object {
                                        "float": "right",
                                      }
                    }
          >
                    Version 
                    <VersionId />
          </div>
          <div>
                    Orginal Design ©2017-18 
                    <a
                              href="https://twitter.com/thatmattwade"
                              rel="noopener noreferrer"
                              target="_blank"
                    >
                              Matt Wade
                    </a>
                     & 
                    <a
                              href="http://icansharepoint.com"
                              rel="noopener noreferrer"
                              target="_blank"
                    >
                              icansharepoint.com
                    </a>
                      Web Development ©2017-18  
                    <a
                              href="https://twitter.com/niegrejoh"
                              rel="noopener noreferrer"
                              target="_blank"
                    >
                              Niels Gregers Johansen
                    </a>
                      
                    <a
                              href="https://www.hexatown.com"
                              rel="noopener noreferrer"
                              target="_blank"
                    >
                              hexatown.com
                    </a>
                       
          </div>
</div>,
        "className": "NavigationFooter",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              style={
                            Object {
                                          "float": "right",
                                        }
              }
>
              Version 
              <VersionId />
</div>,
            <div>
              Orginal Design ©2017-18 
              <a
                            href="https://twitter.com/thatmattwade"
                            rel="noopener noreferrer"
                            target="_blank"
              >
                            Matt Wade
              </a>
               & 
              <a
                            href="http://icansharepoint.com"
                            rel="noopener noreferrer"
                            target="_blank"
              >
                            icansharepoint.com
              </a>
                Web Development ©2017-18  
              <a
                            href="https://twitter.com/niegrejoh"
                            rel="noopener noreferrer"
                            target="_blank"
              >
                            Niels Gregers Johansen
              </a>
                
              <a
                            href="https://www.hexatown.com"
                            rel="noopener noreferrer"
                            target="_blank"
              >
                            hexatown.com
              </a>
                 
</div>,
          ],
          "className": "NavigationFooterContent ms-font-m",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Version ",
                <VersionId />,
              ],
              "style": Object {
                "float": "right",
              },
            },
            "ref": null,
            "rendered": Array [
              "Version ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Orginal Design ©2017-18 ",
                <a
                  href="https://twitter.com/thatmattwade"
                  rel="noopener noreferrer"
                  target="_blank"
>
                  Matt Wade
</a>,
                " & ",
                <a
                  href="http://icansharepoint.com"
                  rel="noopener noreferrer"
                  target="_blank"
>
                  icansharepoint.com
</a>,
                "  Web Development ©2017-18  ",
                <a
                  href="https://twitter.com/niegrejoh"
                  rel="noopener noreferrer"
                  target="_blank"
>
                  Niels Gregers Johansen
</a>,
                "  ",
                <a
                  href="https://www.hexatown.com"
                  rel="noopener noreferrer"
                  target="_blank"
>
                  hexatown.com
</a>,
                "   ",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Orginal Design ©2017-18 ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Matt Wade",
                  "href": "https://twitter.com/thatmattwade",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "Matt Wade",
                "type": "a",
              },
              " & ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "icansharepoint.com",
                  "href": "http://icansharepoint.com",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "icansharepoint.com",
                "type": "a",
              },
              "  Web Development ©2017-18  ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Niels Gregers Johansen",
                  "href": "https://twitter.com/niegrejoh",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "Niels Gregers Johansen",
                "type": "a",
              },
              "  ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "hexatown.com",
                  "href": "https://www.hexatown.com",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "hexatown.com",
                "type": "a",
              },
              "   ",
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
